internal/appconfig/patches.go

```diff
@@ -172,14 +172,22 @@ func patchServices(cfg map[string]any) (map[string]any, error) {
 			return nil, fmt.Errorf("Error processing services: %w", err)
 		}
 
-		for idx, service := range services {
+		var newServices []map[string]any
+		for _, service := range services {
 			service, err := _patchService(service)
 			if err != nil {
 				return nil, err
 			}
-			services[idx] = service
+			if len(service) != 0 {
+				newServices = append(newServices, service)
+			}
+		}
+
+		if len(services) > 0 {
+			cfg["services"] = newServices
```

Comment:
Could be misunderstanding, but shouldn't this be `len(newServices)`?

Reply:
you are right but that left me thinking if we need the length check at all 🤔

Reply:
nice catch! fixed by c484058b